### PR TITLE
Remote: don't print tracebacks for Error exceptions handled downstream

### PR DESCRIPTION
**Pull request by enkore**
(GH wanted to put this into master before)

### CODE REVIEW

File: borg/remote.py [hunk 1]
```diff
@@ -93,8 +94,11 @@ def serve(self):
                             f = getattr(self.repository, method)
                         res = f(*args)
                     except BaseException as e:
-                        logging.exception('Borg %s: exception in RPC call:', __version__)
-                        logging.error(sysinfo())
+                        if not isinstance(e, (Repository.DoesNotExist, Repository.AlreadyExists, Repository.CheckNeeded,
+                                              IntegrityError, PathNotAllowed, Repository.ObjectNotFound,
+                                              InvalidRPCMethod)):
```
Comment (ThomasWaldmann) [inline borg/remote.py]: are these the ones subclassing Error (not ErrorWithTraceback)?

Comment (ThomasWaldmann) [inline borg/remote.py]: and if so, could we just check for that?

Comment (enkore) [inline borg/remote.py]: No, these are the special-cased exceptions handled [here](https://github.com/enkore/borg/blob/issue/792/borg/remote.py#L234). These are the only Errors that the error-handling code in archiver.py:main will see.

Comment (enkore) [inline borg/remote.py]: I'll toss a note in about the relation between these two code blocks

**Comment by enkore** [comment]
I updated this after some more testing and removed some of the exceptions handled by the client, namely all deriving from ErrorWithTraceback, who should (also) print a (remote) traceback.

**Pull request merged**